code/modules/reagents/chemistry/reagents.dm

```diff
@@ -76,6 +82,13 @@ GLOBAL_LIST_INIT(name2reagent, build_name2reagent())
 	var/list/reagent_removal_skip_list = list()
 	///The set of exposure methods this penetrates skin with.
 	var/penetrates_skin = VAPOR
+	// See fermi_readme.dm REAGENT_DEAD_PROCESS, REAGENT_DONOTSPLIT, REAGENT_ONLYINVERSE, REAGENT_ONMOBMERGE, REAGENT_INVISIBLE, REAGENT_FORCEONNEW, REAGENT_SNEAKYNAME
+	var/chemical_flags 
+	//impure chem values (see fermi_readme.dm for more details):
+	var/impure_chem		 = /datum/reagent/impure			// What chemical path is made when metabolised as a function of purity
+	var/inverse_chem_val = 0.2								// If the impurity is below 0.5, replace ALL of the chem with inverse_chem upon metabolising
+	var/inverse_chem	 = /datum/reagent/impure/toxic		// What chem is metabolised when purity is below inverse_chem_val
+	var/failed_chem		 = /datum/reagent/consumable/failed_reaction //what chem is made at the end of a reaction IF the purity is below the recipies purity_min
```

Comment:
Could you use the dmdoc `///` format here?

Reply:
You mean like this? Or should I move the after var comments onto a new line too?